app/backend/src/couchers/models.py

```diff
@@ -213,6 +213,14 @@ class User(Base):
     phone_verification_verified = Column(DateTime(timezone=True), nullable=True, server_default=text("NULL"))
     phone_verification_attempts = Column(Integer, nullable=False, server_default=text("0"))
 
+    # Verified phone numbers should be unique
+    Index(
+        "ix_users_unique_phone",
+        phone,
+        unique=True,
+        postgresql_where=phone_verification_verified != None,
+    ),
```

Comment:
Should `Index`es be moved out of `__table_args__`? There's a few more instances :S

Reply:
Yeah, we could move the rest in a new PR. sqlalchemy documentation is sparse on this metaprogramming style of defining tables.